.github/ISSUE_TEMPLATE/release-upgrade-testing.md

```diff
@@ -0,0 +1,159 @@
+---
+name: Release Demo and Upgrade Testing
+about: |
+  This template can be used to track the upgrade testing of demos from the outgoing stable to
+  the new release, and new release from scratch before officially marking the new release as stable.
+title: "chore(tracking): Test demos on nightly versions for YY.M.X"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the filename.
+-->
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
```

Comment:
note: This needs updating (and in `pre-release-upgrade-testing.md`).

```suggestion
<!--
    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
    This was created by an issue template: https://github.com/stackabletech/demos/issues/new/choose.
-->
```

On the other hand, we might just remove it entirely.